packages/pieces/apps/src/lib/trello/actions/create-card.ts

```diff
@@ -0,0 +1,53 @@
+import { createAction, httpClient, HttpRequest, HttpMethod, Property } from '@activepieces/framework';
+import { trelloCommon } from '../common';
+import { TrelloCard } from '../common/props/card';
+
+export const createCard = createAction({
+	name: 'create_card',
+    displayName:'Create Card',
+    description: 'Create a new card in Trello',
+	props: {
+        authentication: trelloCommon.authentication,
+        token: trelloCommon.token,
+        listId: Property.ShortText({

```

Comment:
We can remove this step for the users, by creating dropdown that loads lists when the user connect their account.

Check this example in Google Drive. (https://github.com/activepieces/activepieces/blob/main/packages/pieces/apps/src/lib/google-drive/action/create-new-folder.ts#L26)